docs/how_to/use_estimator.rst

```diff
@@ -0,0 +1,265 @@
+###############################################
+Compute an expectation value with ``Estimator``
+###############################################
+
+This guide shows how to get the expected value of an observable for a given quantum circuit with the :class:`~qiskit.primitives.Estimator` primitive.
+
+.. note::
+
+    While this guide uses Qiskit’s reference implementation, the ``Estimator`` primitive can be run with any provider using :class:`~qiskit.primitives.BackendEstimator` .
+    
+    .. code-block::
+
+        from qiskit.primitives import BackendEstimator
+        from qiskit_provider import QiskitProvider
+
+        provider = QiskitProvider()
+        backend = provider.get_backend('backend_name')
+        estimator = BackendEstimator(backend)
+
+    There are some providers that implement primitives natively (see `this page <http://qiskit.org/providers/#primitives>`_ for more details).
+
+
+Initialize observable
+=====================
+
+The first step is to define the observable whose expected value you want to compute. This observable can be any ``BaseOperator``, like the operators from :mod:`qiskit.quantum_info`.
+Among them it is preferable to use :class:`~qiskit.quantum_info.SparsePauliOp`.
+
+.. testcode::
+
+    from qiskit.quantum_info import SparsePauliOp
+
+    observable = SparsePauliOp(["II", "XX", "YY", "ZZ"], coeffs=[1, 1, -1, 1])
+
+Initialize quantum circuits
```

Comment:
Is the "circuits" plural intentional

Reply:
Yes. You can actually define several circuits for one run of both `Sampler` and `Estimator`. In the case of `Estimator`  you can also pass a list of observables. 

I see that I left the "Initialize observable" title in singular though. 😅

Reply:
and the example is with a single circuit.

Reply:
Solved by 51a1260fea0cfa9b437af39918555af85fc520d8